### DISCUSSION TITLE
Chained Member Access Expression

### DISCUSSION THREAD

**Discussion by aesteve** [dotnet/roslyn#68640] [Q&A] [answered]
Hello and thanks for your work.

I'm using the Syntax API to parse the following `InvocationExpressionSyntax`:   

```c#
 _theProperty.MemberOne(param => "something").MemberTwo(param => "something else");
```

Ideally, I would like to capture the Syntax node matching the content of the second lambda expression.

No problem to get the content of the first one, it's almost direct, by accessing the `Expression`.
But for the second, no clue.

In all descendant nodes of the expression syntax, the only one where `MemberTwo` seems to appear is an `IdentifierNameSyntax`.

I must be missing something obvious (sorry I'm completely new to Roslyn), could anyone give me a pointer to what I'm missing? Thank you.

**Comment by CyrusNajmabadi** [comment, marked as answer]
I recommend using sharplab.io.  It has a mode that will show you the syntax tree produced.

**Reply by aesteve** [reply]
Exactly the pointer I was looking for. Thanks a lot this led me on the right track.

And to my original question, the expression itself is what I am looking for (defined from right to left) not its decendantNodes.
